File: Python/BAEKJOON/BRONZE Ⅲ/2720 세탁소 사장 동혁.py
for _ in range(int(input())):
    C = int(input())

    quarter, C = divmod(C, 25)
    dime, C = divmod(C, 10)
    nickel, penny = divmod(C, 5)

    print("%d %d %d %d" % (quarter, dime, nickel, penny))

File: javascript/BAEKJOON/BRONZE Ⅲ/2720 세탁소 사장 동혁.js
var input = require('fs').readFileSync(__dirname+'/test.txt').toString().trim().split('\n');
//var input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

for (let i = 1; i <= parseInt(input[0]) ; i++) {

    let C = parseInt(input[i]);

    const quarter = Math.floor(C / 25);
    C %= 25;

    const dime = Math.floor(C / 10);
    C %= 10;

    const nickel = Math.floor(C / 5);
    C %=  5;

    const penny = C;

    console.log(`${quarter} ${dime} ${nickel} ${penny}`);
}